manager/src/main/java/com/redhat/service/smartevents/manager/workers/resources/BridgeWorker.java

```diff
@@ -75,6 +80,9 @@ public Bridge createDependencies(Work work, Bridge bridge) {
         // We don't need to wait for the Bridge to be READY to create the Error Handler.
         createErrorHandlerProcessor(bridge);
 
+        // Create DNS record
+        dnsService.get().createDnsRecord(bridge.getId());
```

Comment:
Did you want to make your `DnsService` implementations `@ApplicationScoped`?

I think the use of `get()` will return a new instance on each invocation as they have the default scope.

Reply:
Wops, that's a leftover